src/verilog.l

```diff
@@ -106,6 +113,18 @@ vnum    {vnum1}|{vnum2}|{vnum3}|{vnum4}|{vnum5}
 
 <INITIAL>.|\n           { BEGIN STATE_VERILOG_RECENT; yyless(0); }
 
+  /************************************************************************/
+  /* File paths */
+<FILEPATH>{
+  {ws}            { FL_FWD; FL_BRK; }
```

Comment:
Will need a test for this too; see the similar _bad examples for EOF.

```suggestion
<FILEPATH>{
 <<EOF>>         { FL; yylval.fl->v3error("EOF in unterminated file path");
                          yyleng = 0; yy_pop_state(); FL_BRK; yyterminate(); }
  {ws}            { FL_FWD; FL_BRK; }
```

Reply:
Are you sure the // comment isn't passing through and that's the ; that ends the lex?  Your lex rule doesn't strip comments (which I do think is correct BTW)